packages/hash/frontend/src/lib/subgraph.ts

```diff
@@ -0,0 +1,231 @@
+import {
```

Comment:
cc: @yusufkinatas @nathggns @luisbettencourt @CiaranMn 

As the Subgraph becomes more prevalent in the frontend code, I suspect we'll want more helper methods to interact with it. I have created this file to hold them while we flesh such things out. Tagging you so you know they're here, hopefully it'll save you frustration